cluster/addons/cluster-monitoring/influxdb/heapster-controller.json

```diff
@@ -0,0 +1,54 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "ReplicationController",
+  "metadata": {
+    "labels": {
+      "name": "heapster",
+      "kubernetes.io/cluster-service": "true"
+    },
+    "name": "monitoring-heapster-controller"
+  },
+  "spec": {
+    "replicas": 1,
+    "selector": {
+      "name": "heapster"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "heapster",
+          "kubernetes.io/cluster-service": "true"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "image": "gcr.io/google_containers/heapster:v0.11.0",
+            "name": "heapster",
+            "env": [
+              {
+                "name": "FLAGS",
+                "value": "--poll_duration=2m --stats_resolution=1m --sink influxdb:http://$MONITORING_INFLUXDB_SERVICE_HOST:$MONITORING_INFLUXDB_PORT_8086_TCP_PORT"
```

Comment:
Why use env variables? Why not use DNS? This could be `http://monitoring-influxdb:80`.
The advantage with dns is that all these services can be started in parallel.